.classpath

```diff
@@ -7,5 +7,7 @@
 	<classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/TalonSRXLibJava.jar"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
```

Comment:
Whoa. This right here is why we've been getting runtime errors on the RoboRio. See
[README.txt](https://github.com/frc2399/2017-Season/files/764400/README.txt) from the talon library.